python/ray/scripts/scripts.py

```diff
@@ -588,7 +588,8 @@ def rsync_up(cluster_config_file, source, target, cluster_name):
 @click.option(
     "--port-forward", required=False, type=int, help="Port to forward.")
 @click.argument("script", required=True, type=str)
-@click.argument("script_args", required=False, type=str, nargs=-1)
+@click.argument(
+    "script_args", required=False, type=click.UNPROCESSED, nargs=-1)
```

Comment:
I think `nargs` works, but this could also be implemented as an argument requiring single string like `--script_args="arg1 arg2"`. Single string argument seems a little less error-prone, but I the current solution might be more usable, so I don't feel strongly either way.

Reply:
Yeah that's a good point; let me change it.